code/datums/uplink_item.dm

```diff
@@ -580,7 +580,7 @@ var/list/uplink_items = list()
 	desc = "When inserted into a personal digital assistant, this cartridge gives you four opportunities to detonate PDAs of crewmembers who have their message feature enabled. \
 	The concussive effect from the explosion will knock the recipient out for a short period, and deafen them for longer. It has a chance to detonate your PDA."
 	item = /obj/item/weapon/cartridge/syndicate
-	cost = 6
+	cost = 2
```

Comment:
Its because demo gets unused, and in our game environment (you can yell "PDA bomb" without getting bwoinked), its easily countered with meta after its first use if the crew is proactive.